app/javascript/avalon-api/index.js

```diff
@@ -0,0 +1,32 @@
+/**
+ * Generic function to handle fetch() errors
+ * @param  {object} response fetch() Response object
+ * @return {Promise} response fetch() Response object
+ */
+function handleErrors(response) {
+  if (!response.ok) {
+    // Identifies in the browser console where error is happening
+    throw Error(response.statusText);
+  }
+  return response;
+}
+
+/**
+ * Fetch a media object manifest object
+ * @param  {string} id Media object id
+ * @return {Promise}
+ */
+export const fetchManifest = id => {
+  return fetch(`http://localhost:3333/concern/generic_works/${id}/manifest`, {
```

Comment:
For PoC it's probably ok to have hardcoded path here, but I think we should make the `host:port` generic so it works for everyone, something like `location.host` maybe.

Reply:
@phuongdh Good call.  I was wondering about this.  Should I update this now to just get it right?

Reply:
Updating now to `location.origin`